Patch from https://github.com/pytorch/pytorch/pull/45339
Patch added to EasyBuild by Simon Branford, University of Birmingham
diff --git a/aten/src/ATen/native/cpu/CopyKernel.cpp b/aten/src/ATen/native/cpu/CopyKernel.cpp
index 3dbd52aa19055..bca831dc55e2c 100644
--- a/aten/src/ATen/native/cpu/CopyKernel.cpp
+++ b/aten/src/ATen/native/cpu/CopyKernel.cpp
@@ -17,6 +17,8 @@ static void copy_kernel(TensorIterator& iter, bool non_blocking) {
       cpu_kernel(iter, [=](at::Half a) -> at::Half { return a; });
     } else if (dtype == ScalarType::BFloat16) {
       cpu_kernel(iter, [=](at::BFloat16 a) -> at::BFloat16 { return a; });
+    } else if (dtype == ScalarType::ComplexHalf) {
+      cpu_kernel(iter, [=](c10::complex<at::Half> a) -> c10::complex<at::Half> { return a; });
     } else if (isQIntType(dtype)) {
       AT_DISPATCH_QINT_TYPES(dtype, "copy_kernel", [&] {
         cpu_kernel_vec(
